perspective.el

```diff
@@ -408,23 +378,28 @@ EVENT is the click event triggering this function call."
   (interactive "e")
   (persp-switch (format "%s" (car (posn-string (event-start event))))))
 
+(defun persp-mode-line ()
+  "Use this in the mode-line to show up-to-date value of `persp-modestring'."
```

Comment:
`persp-modestring` doesn't refer to anything anymore. I'd write this docstring as "Return the string displayed in the modeline representing the perspectives."

Similarly, update other docstrings that refer to `persp-modestring`.

Reply:
I've updated the docstrings refering to `persp-modestring`,